server/api/support.py

```diff
@@ -175,7 +175,8 @@ def get_election(election_id: str):
             for jurisdiction in election.jurisdictions
         ],
         rounds=[
-            dict(id=round.id, ended_at=round.ended_at) for round in election.rounds
+            dict(id=round.id, endedAt=round.ended_at, roundNum=round.round_num)
```

Comment:
As noted in the commit message, changed `ended_at` to `endedAt` in the JSON response for consistency with all our other camelCase JSON fields